kuma/api/v1/tests/test_views.py

```diff
@@ -225,7 +226,20 @@ def test_whoami_anonymous(client, api_settings, timezone):
     response = client.get(url, HTTP_HOST=api_settings.BETA_HOST)
     assert response.status_code == 200
     assert response['content-type'] == 'application/json'
-    assert response.json() == {
+    data = response.json()
+
+    # pytest restrictions on using the db during tests means that
+    # we don't know what flags, switches and samples we expect to
+    # receive in the response. This test assumes that waffle works
+    # correctly, so all we are doing here is validating that the
+    # response includes dicts in the expected places.
+    assert type(data['waffle']['flags']) == dict
```

Comment:
So the basic problem was that I couldn't figure out a way to get test data into those flags. Possibly that was because I was initializing too soon. Now that I call get_all() later on, maybe I can get it to work.

But since I couldn't get any data, I was just verifying that the endpoint sends waffle.flags etc, and furthermore that waffle.flags is a dict and not a string, for example.

I'll make more attempt to get some actual test waffle data working. If that fails, I'll convert to isinstance sintead of using type()